src/plugins/embeddable/public/embeddable_panel/panel_actions/customize_panel_action/customize_panel_editor.test.tsx

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import { overlayServiceMock } from '@kbn/core-overlays-browser-mocks';
+import { themeServiceMock } from '@kbn/core-theme-browser-mocks';
+import { __IntlProvider as IntlProvider } from '@kbn/i18n-react';
+import '@testing-library/jest-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+
+import { Filter } from '@kbn/es-query';
+import { ViewMode } from '../../../../common';
+import { FilterableEmbeddable, IEmbeddable } from '../../../lib';
+import {
+  ContactCardEmbeddable,
+  ContactCardEmbeddableFactory,
+  ContactCardEmbeddableInput,
+} from '../../../lib/test_samples';
+import { EditPanelAction } from '../edit_panel_action/edit_panel_action';
+import { CustomizePanelAction } from './customize_panel_action';
+import { CustomizePanelEditor } from './customize_panel_editor';
+
+const overlays = overlayServiceMock.createStartContract();
+const theme = themeServiceMock.createStartContract();
+const editPanelActionMock = { execute: jest.fn() } as unknown as EditPanelAction;
+
+const mockEmbeddableFactory = new ContactCardEmbeddableFactory((() => null) as any, {} as any);
+const customizePanelAction = new CustomizePanelAction(overlays, theme, editPanelActionMock);
+customizePanelAction.execute = jest.fn();
+
+const DEFAULT_PANEL_TITLE = 'Panel title';
+
+const createEmbeddable = async (
+  initialInput?: Partial<ContactCardEmbeddableInput>
+): Promise<ContactCardEmbeddable> => {
+  return await mockEmbeddableFactory.create({
+    id: '20',
+    firstName: 'Bilbo',
+    lastName: 'Baggins',
+    title: DEFAULT_PANEL_TITLE,
+    ...initialInput,
+  });
+};
+
+const DEFAULT_PROPS = {
+  timeRangeCompatible: true,
+  onClose: jest.fn(),
+  onEdit: jest.fn(),
+};
+
+describe('panel title / description', () => {
+  test('does not render if in view mode', async () => {
+    const mockEmbeddable = await createEmbeddable({ viewMode: ViewMode.VIEW });
+    render(
+      <IntlProvider locale="en">
+        <CustomizePanelEditor {...DEFAULT_PROPS} embeddable={mockEmbeddable} />
+      </IntlProvider>
+    );
+
+    const customizePanelForm = await screen.findByTestId('customizePanelForm');
+    const titleDescriptionComponent = screen.queryByTestId('customEmbeddableTitleComponent');
+    expect(customizePanelForm).not.toContainElement(titleDescriptionComponent);
+  });
+
+  test('title input receives focus when `titleFocus` is `true`', async () => {
```

Comment:
Awesome looking RTL tests, nice work! This will cause some fun conflicts with [this](https://github.com/elastic/kibana/blob/0ebe79f6c41dddb47f14e89ebee7507186af0dbd/src/plugins/presentation_panel/public/panel_actions/customize_panel_action/customize_panel_editor.test.tsx)